src/components/Home/HomeMap.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { Menu, Popup, Icon, Label } from 'semantic-ui-react';
+import GoogleMapReact from 'google-map-react';
+
+const style = {
+  borderRadius: 0,
+  border: '1px solid white',
+  opacity: 0.7,
+  backgroundColor: 'grey',
+  color: 'white'
+};
+
+const AnyReactComponent = ({ opened, onClose, labelClass, mouseEnter, mouseLeave, clicked, date, upVote, downVote, city, text, color }) => (
+  <Popup
```

Comment:
`AnyReactComponent`? I feel like this should be named to something more specific. Also should be placed in a separate file.